core/tests.py

```diff
@@ -6,25 +6,22 @@
 """
 
 from django.test import TestCase
-from core.models import UserMetrics, UserProfile,User, GlobalMetrics,MostKnown
+from core.models import UserMetrics, UserProfile, GlobalMetrics, MostKnown
```

Comment:
Removed User obj, because it didn't exist in core.models (its a django default user model)